### DISCUSSION TITLE
blog/posts/problem-k-th-smallest-prime-fraction/

### DISCUSSION THREAD

**Discussion by giscus** [KTS-o7/KTS-o7.github.io#6] [General]
# blog/posts/problem-k-th-smallest-prime-fraction/

Problem Statement Link - Problem 786 Question You are given a sorted integer array arr containing 1 and prime numbers, where all the integers of arr are unique. You are also given an integer k.
For every i and j where 0 &lt;= i &lt; j &lt; arr.length, we consider the fraction arr[i] / arr[j].
Return the kth smallest fraction considered. Return your answer as an array of integers of size 2, where answer[0] == arr[i] and answer[1] == arr[j].

https://kts-o7.github.io/blog/posts/problem-k-th-smallest-prime-fraction/

**Comment by DeathStroke19891** [comment]
An explanation for the follow up would have been nice, especially for us noobs here. Great effort tho :)
